File: php/counter.php
<html><body>
<h3>Counter example</h3>

<?php 

echo 'This page has been loaded ';

$t = $db["phpCount"];
$o = $t->findOne();
if( !$o )
  $o = array( "count" => 0 );
$o["count"]++;
$t->save($o);

echo $o["count"];
echo " times";
?> 

<p>
<a href="/admin/doc?f=/php/counter.php">[Code]</a>
</body></html>

File: py/counter.py
n=db.nLoads.findOne() or {'count':0}
n.count = n.count + 1
db.nLoads.save(n)

print "<html><body>This page has been loaded "
print n.count
print " times. "
print '<a href="/admin/doc?f=/py/counter.py">[code]</a>'
print "<p>"





